### DISCUSSION TITLE
About recommended bandwidth for Hubs

### DISCUSSION THREAD

**Discussion by jasoncks** [mozilla/hubs#3289] [Hubs Cloud Questions] [answered]
Do Hubs Cloud have any information or suggestions in recommended network bandwidth?

  | Mobile (iOS,Android) | Mobile (iOS,Android) | PC(Windows,Mac) | PC(Windows,Mac) | VR Devices | VR Devices 
-- | -- | -- | -- | -- | -- | --
-- | Download(kbps) | Upload(kbps) | Download(kbps) | Upload(kbps) | Download(kbps) | Upload(kbps)
View in Room |   |   |   |   |   |  
View in Lobby |   |   |   |   |   |  
Play Video in Rooms |   |   |   |   |   |  
Voice Chat in Rooms |   |   |   |   |   |  
Screen Share in Rooms |   |   |   |   |   |  

Thank you

**Comment by emclaren** [comment, marked as answer]
This is a good question, however, its also one that is quite challenging to answer. The amount of bandwidth needed varies wildly based on the composition of the scene, the other avatars in the space, and the type of media shared. 

I will surface this to our team. Perhaps in the future it would be possible to establish baseline recommendations, for the time being though we are unable to provide definitive information related to bandwidth required across different devices because it is so varied based on the room content.

**Reply by robinkwilson** [reply]
Marking as answered for now, unmark as answered if you need this to be reopened.
